package.json

```diff
@@ -26,7 +26,8 @@
     "Mikael Lepistö <mikael.lepisto@vincit.com> (https://github.com/elhigu)",
     "Matthew McEachen <matthew-objection@photostructure.com> (https://github.com/mceachen)",
     "Jürg Lehni <juerg@scratchdisk.com> (https://github.com/lehni)",
-    "Igor Savin <kibertoad@gmail.com> (https://github.com/kibertoad)"
+    "Igor Savin <kibertoad@gmail.com> (https://github.com/kibertoad)",
+    "Guillaume Robin<robinguillaume.pro@gmail.com> (https://github.com/cesumilo)"
```

Comment:
@cesumilo for now this list is quite short and contains only people who wrote rather complex features or fixes or are/were maintainers. Would you be OK if we waited to add you until we're certain we will work together in this way?

Reply:
Also, since none of the dependencies changed, `package-lock.json` shouldn't actually change?

Reply:
Oh yes sorry 😵‍💫, it was to experiment what it meant for npm. I forgot to remove it.